### ISSUE TITLE
Change title of Meetings to All Meetings

### ISSUE THREAD

**Issue by Grateful-Viv** [code4recovery/tsml-ui#123]
I see it switches when selecting type active meetings to "Active Meetings", selecting type online to "Online Meetings", etc. Can the default title for showing all meetings (includes inactive), be changed to All Meetings?

**Comment by joshreisner** [comment]
When filters are applied, title pieces are appended to the base of "Meetings." The base can customized to say something like "AA Meetings." 

When the next version of TSML ships, it should be possible to add the following to a child theme to make this customization:

```
$tsml_ui_config['strings']['en']['meetings'] = 'AA Meetings';
```

However that's not quite what you're looking for, because the base is present in every variation, so that would become "Midday AA Meetings" when the "Midday" time filter is applied.

What you're looking for is a fallback phrasing when no filters are applied. I don't see a pressing enough need for this currently to add that complexity.

**Comment by joshreisner** [comment]
I don't think this would change the outcome, but I want to be sure I understood the requested logic correctly. Would you want the word "All" in any/all of the following situations?

* All Midday Meetings
* All LGBT Meetings
* All Meetings in Lewes
* All Monday Meetings in Lewes
* All In-person Meetings in Lewes

I'm assuming based on the request that you definitely don't want it here:

* Active Meetings in Lewes

Basically: would it be fallback phrasing when no filters are applied, or in all cases when Active is not applied?
